protocol/directory.go

```diff
@@ -31,15 +36,18 @@ func InitDirectory(epDeadline merkletree.TimeStamp, vrfKey vrf.PrivateKey,
 	return d
 }
 
-func (d *ConiksDirectory) Update() {
+func (d *ConiksDirectory) Update(up *UpdatePolicies) {
```

Comment:
Ok yeah, I saw that, which is why I made my comment. The point of the epoch deadline was to give the clients a firm time for when to expect the next STR (hence the name deadline and the `TimeStamp` type). If it's just a time interval, clients have to compute for themselves if the server met the deadline, while if it's an actual time stamp, barring clock synchronization issues, the server is forced to publish the next STR by that time and clients can easily verify that the server has done this.